dungeon/chain.json

```diff
@@ -0,0 +1,59 @@
+{
+  "chain_name": "dungeon",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.cryptodungeon.org",
+  "pretty_name": "Dungeon Chain",
+  "chain_id": "dungeon-1",
+  "assets": [
```

Comment:
assets aren't repeated in the chain.json file

Reply:
updated

Reply:
Why do you have an assets array here?